pkgs/desktops/arcan/arcan/default.nix

```diff
@@ -106,7 +106,6 @@ stdenv.mkDerivation rec {
     ./000-openal.patch
     ./001-luajit.patch
     ./002-libuvc.patch
-    ./003-freetype.patch
```

Comment:
Why removing this?

Reply:
It was failing to apply, and it's building without. It seems `arcan` changed the handling of freetype to a `find_package` based workflow that works out of the box.

Reply:
Okay then.